libraries/chain/controller.cpp

```diff
@@ -402,11 +402,13 @@ struct controller_impl {
       }
    }
 
-   void dmlog_applied_transaction(const transaction_trace_ptr& t) {
+   void dmlog_applied_transaction(const transaction_trace_ptr& t, const signed_transaction *trx = nullptr) {
```

Comment:
Not a big deal, can `const signed_transaction *trx` be changed to `const signed_transaction* trx` to be consistent with existing style?

Better yet, not make this as a default argument. We try to avoid default arguments unless plain arguments cause big changes, so we know every calling place has been looked at.

Reply:
Actually, I was trying to keep the changes minimal, otherwise I have to add `nullptr` to the six other call locations, but I'm ok doing it if you think it is preferable.

Reply:
I did the whitespace update btw.